tests/Unit/UseCases/FeeChange/FeeChangeUseCaseTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\MembershipContext\Tests\Unit\UseCases\FeeChange;
+
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use WMDE\Fundraising\MembershipContext\Tests\TestDoubles\FailingFeeChangeRepository;
+use WMDE\Fundraising\MembershipContext\Tests\TestDoubles\SucceedingFeeChangeRepository;
+use WMDE\Fundraising\MembershipContext\Tests\TestDoubles\ThrowingFeeChangeRepository;
+use WMDE\Fundraising\MembershipContext\UseCases\FeeChange\FeeChangeRequest;
+use WMDE\Fundraising\MembershipContext\UseCases\FeeChange\FeeChangeResponse;
+use WMDE\Fundraising\MembershipContext\UseCases\FeeChange\FeeChangeUseCase;
+
+#[CoversClass( FeeChangeUseCase::class )]
+#[CoversClass( FeeChangeRequest::class )]
+#[CoversClass( FeeChangeResponse::class )]
+class FeeChangeUseCaseTest extends TestCase {
+	public function testFeeChangeExistsReturnsTrue(): void {
+		$useCase = new FeeChangeUseCase( new SucceedingFeeChangeRepository() );
+
+		$this->assertTrue( $useCase->feeChangeExists( 'uuid' ) );
```

Comment:
I was a bit confused by the `'uuid'` string. This test shows that we really don't care and don't validate the UUID and just assume that it follows the UUID format. Which is totally fine and pragmatic for our use case. But could you define and use a `TEST_UUID` constant in this test instead of `'uuid'`?